src/main/java/org/spongepowered/api/entity/vehicle/minecart/MinecartLike.java

```diff
@@ -90,4 +85,23 @@
     default Value.Mutable<Vector3d> derailedVelocityModifier() {
         return this.requireValue(Keys.DERAILED_VELOCITY_MODIFIER).asMutable();
     }
+
+    /**
+     * {@link Keys#MINECART_BLOCK_OFFSET}
+     *
+     * @return The represented block's offset of the minecart
+     */
+    default Value.Mutable<Integer> minecartBlockOffset() {
```

Comment:
`blockOffset` but should be back in `BlockOccupiedMinecart`

Reply:
https://github.com/SpongePowered/Sponge/blob/1d33b3c3daf2fbcee44554d128deec16c11aa96e/src/main/java/org/spongepowered/common/data/provider/entity/AbstractMinecartData.java#L64-L67